Constitution.md

```diff
@@ -171,17 +171,35 @@ The Committee may appoint as Honorary President and Vice-President, any persons
 2. Notice of such meetings must be given by email at least seven days beforehand, together with the Agenda.
 3. An Extraordinary General Meeting (EGM) may be called by 5% of the membership of the Society by application to the Secretary. An EGM shall be called as specified on the application, but at least 7 days’ notice shall be given.
 4. Quorum of such meetings shall be set at twice the Committee size, plus one.
-5. Only full members, not Friends of the Society, should be counted during any AGM or EGM or in the process of calling for an EGM
+5. Only full members, not Friends of the Society, are to be considered "eligible voters". Only eligible voters should be counted during any AGM or EGM or in the process of calling for an EGM.
 
 ## Elections
 
 1. Elections shall be held at the Annual General Meeting to elect officers of the Committee for the following session.
 2. There shall be provision for by-elections for positions unfilled at the AGM to be held at an EGM in October.
 3. An EGM must be called if the held Committee positions are not in line with Bristol SU regulations.
-4. Any restrictions on who can stand must be stated –note that such restrictions must not contravene the Code of Conduct.
+4. Any restrictions on who can stand must be stated – note that such restrictions must not contravene the Code of Conduct.
 5. Nominations must be made in accordance with instruction officially communicated by the Committee.
-6. The voting system shall be a single vote for each position, with a simple majority.
-7. The Chairperson of the General Meeting to have the casting vote in the event of a tie, having not voted previously.
+6. Voting must be by secret, secure ballot either through paper or electronic means.
+7. The default system of voting shall be First Past the Post ("FPTP"), for ease of counting only. Where FPTP is used as the system of voting, it shall be defined as follows.
+   1. Eligible voters shall cast (up to) the same number of votes as there are positions to be filled.
+   2. Each vote shall be a mark next to the selected candidate(s).
```

Comment:
7.2 This may be needlessly proscriptive - last year the voting was carried out by writing the name of the candidate on a scrap of paper. This could be an issue if ballot sheets have not been created prior to the vote. What was the intention of this subclause? is it possible it could be reworded?

Reply:
I've changed this to ``Each vote shall indicate the selected candidate(s)``